etc/input.conf

```diff
@@ -57,17 +58,17 @@
 #Ctrl+RIGHT  no-osd sub-seek  1         # seek to the next subtitle
 #Ctrl+Shift+LEFT sub-step -1            # change subtitle timing such that the previous subtitle is displayed
 #Ctrl+Shift+RIGHT sub-step 1            # change subtitle timing such that the next subtitle is displayed
-#Alt+left  add video-pan-x  0.1         # move the video right
-#Alt+right add video-pan-x -0.1         # move the video left
-#Alt+up    add video-pan-y  0.1         # move the video down
-#Alt+down  add video-pan-y -0.1         # move the video up
+#Alt+LEFT  script-binding positioning/pan-x -0.2 # pan to the right
+#Alt+RIGHT script-binding positioning/pan-x  0.2 # pan to the left
+#Alt+UP    script-binding positioning/pan-y -0.2 # pan down
+#Alt+DOWN  script-binding positioning/pan-y  0.2 # pan up
 #Alt++     add video-zoom   0.1         # zoom in
 #ZOOMIN    add video-zoom   0.1         # zoom in
 #Alt+-     add video-zoom  -0.1         # zoom out
 #ZOOMOUT   add video-zoom  -0.1         # zoom out
-#Ctrl+WHEEL_UP   add video-zoom  0.1    # zoom in
-#Ctrl+WHEEL_DOWN add video-zoom -0.1    # zoom out
-#Alt+BS set video-zoom 0; set panscan 0; set video-pan-x 0; set video-pan-y 0 # reset zoom and pan settings
+#Ctrl+WHEEL_UP   script-binding positioning/cursor-centric-zoom  0.1 # zoom in towards the cursor
+#Ctrl+WHEEL_DOWN script-binding positioning/cursor-centric-zoom -0.1 # zoom out towards the cursor
```

Comment:
Does it need to be changed? Could we use Ctrl+Alt modifier for new behavior? Also could update numpad bindings.

Reply:
Is there a reason not to have cursor-centric-zoom on the wheel? It is more convenient, also according to nanahi ctrl+wheel "makes "pinch to zoom" with touchpads and touchscreens work out of box on Windows, since by default applications receive these key inputs for pinch gesture.", so it needs to be bind to ctrl+wheel to work with pinch gestures by default.

I have no idea what numpad bindings have to do with cursor-centric-zoom, the numpad is not a cursor.

Reply:
> Is there a reason not to have cursor-centric-zoom on the wheel? It is more convenient, also according to nanahi ctrl+wheel "makes "pinch to zoom" with touchpads and touchscreens work out of box on Windows, since by default applications receive these key inputs for pinch gesture.", so it needs to be bind to ctrl+wheel to work with pinch gestures by default.

Ok, if you think it is more convenient this way. I think it is still valuable to control zoom without messing up video positioning completely on each step. But I guess by default it can be cursor centric.

Note that same issues applies, if lua is disabled in mpv build, it probably should fallback to normal video-zoom.

Reply:
You can still use the Alt key bindings for that, we didn't even have Ctrl+wheel until recently.

I removed the Alt+Arrow bindings so you do want to implement a fallback system just for this? I guess we can define alternative bindings with some keyword that input.c can recognize?